index.js

```diff
@@ -43,12 +44,14 @@ function parseChangelog (options, callback) {
  * Internal parsing logic.
  *
  * @param {options} options - options object
- * @param {string} options.filePath - path to changelog file
+ * @param {string=} options.filePath - path to changelog file (or you can use options.text)
+ * @param {string=} options.text - changelog text (or you can use options.filePath)
```

Comment:
I'm not familiar with this notation: `string=`

Reply:
IIRC it means the parameter is optional, if there were a default value it would go to the right of the =. I may have put it in the wrong place though, I'll check tomorrow

Reply:
The notation for defaults in jsdoc is:

```
* @param {string} [param="default"] - description
```

For optional params, it's just the brackets:

```
* @param {string} [param] - description
```